File: LeetCode_2351/Main.cpp
#include<bits/stdc++.h>
using namespace std;

class Solution {
public:
    char repeatedCharacter(string s) {
        int cnt[26] = {0};
        for (int i = 0; i < s.size(); i++) {
            if (cnt[s[i] - 'a']) {
                return s[i];
            }
            cnt[s[i] - 'a']++;
        }
        return ' ';
    }
};

File: LeetCode_2351/Main.java
package LeetCode_2351;

import java.util.Arrays;

class Solution {
    public char repeatedCharacter(String s) {
        int n = s.length();
        int[] cnt = new int[26];
        Arrays.fill(cnt, 0);
        for (int i = 0; i < n; i++) {
            if (cnt[s.charAt(i) - 'a'] > 0) {
                return s.charAt(i);
            }
            cnt[s.charAt(i) - 'a']++;
        }
        return ' ';
    }
    
}

public class Main {
    public static void main(String[] args) {
        
    }
}
